web/registry/tests/test_integration.py

```diff
@@ -84,17 +82,39 @@ def test_web_registry_integration(self):
         )
         full_id = f"SimpleComponent=={TESTING_BRANCH_NAME}"
         self.assertEqual(
-            flat_comp_spec[full_id]["class_name"], "SimpleComponent",
+            flat_comp_spec[full_id]["class_name"],
+            "SimpleComponent",
         )
         self.assertEqual(flat_comp_spec[full_id]["repo"], "AgentOSRepo")
 
-        # TODO - FIXME
-        # Test adding a ComponentRun
-        # param_set = {"SimpleComponent": {"add_to_init_member": {"i": 10}}}
-        # comp_run = simple_component.run("SimpleComponent", param_set)
-        # web_registry.add_run_command_spec(comp_run.run_command)
+        # Test adding a RunCommand
+        param_set = {"SimpleComponent": {"add_to_init_member": {"i": 10}}}
+        comp_run = simple_component.run("add_to_init_member", param_set)
+        run_cmd = comp_run.run_command
+        print("trying to add run_command_spec to web_registry:")
+        pprint.pprint(run_cmd.to_spec())
```

Comment:
I'm not sure how you feel, but I have a vague intuition that we shouldn't leave informational print statements in tests (see prints below as well).  

Relatedly: we should probably clean up the informational prints a bit in the actual runtime at some point too. 🤪

Reply:
I've deleted a bunch and converted some into logging statements, mostly at the DEBUG level. I've held off removing all of the print statements in the core component creation process (the ones about adding dependencies) since i think they are currently kind of nice for demo purposes.